src/ontology/uberon-edit.obo

```diff
@@ -222728,7 +222728,6 @@ def: "A hypothalamic region in rodents that is homologous to the lateral tuberal
 comment: While lateral tuberal nucleus is often used interchangeably with tuberal nucleus in rodents, lateral tuberal nucleus is specific to humans and higher primates. In some cases, lateral tuberal nucleus is thought to be part of the tuberal nucleus in rodents, and could possibly be homologous to human tuberal region instead. {xref="PMID:29976824", xref="neuronames:1890", xref="PMID:1362279"}
 synonym: "lateral tuberal nucleus (sensu Rodentia)" BROAD [neuronames:1890]
 synonym: "lateral tuberal nucleus (sensu Rodentia)" EXACT [PMID:29976812]
-xref: DMBA:16271
```

Comment:
Just for the record: the proper way to workaround the issue caused by the problematic axiom in the DMBA bridge (which is something that ultimately must be fixed [upstream](https://github.com/obophenotype/ABA_Uberon/issues/36), the problem is _not_ in Uberon) is not to remove that cross-reference (the mapping here is correct) but to manually remove DMBA:16271 from the `src/ontology/bridge/uberon-bridge-to-dmba.owl` file, after that file has been refreshed from its original online location.

Reply:
My personal preference would be to literally freeze the bridge in the makefile by commenting it out, but I guess I will try to put the screws on the upstream issue instead. Thanks for linking it!

Reply:
To be clear the fix must happen in [this repository](https://github.com/brain-bican/developing_mouse_brain_atlas_ontology/issues/1); I've linked the issue in the old repository only because it has a proper description of the problem, contrary to the new one that just states what the fix must be.